mysuppressions.xml

```diff
@@ -6,5 +6,5 @@
 
 <suppressions>
     <suppress files="[/\\]src[/\\]test[/\\]java[/\\][a-zA-Z0-9]*" checks="[a-zA-Z0-9]*"/>
-    
+    <suppress checks="MagicNumberCheck" files="gui.*"/>
```

Comment:
On another note, shouldnt this line contain `files="gui/*"`? I am fairly sure that the `.` character is also a wildcard, but just for 1 character.